app/templates/briefs/application_submitted.html

```diff
@@ -36,14 +36,14 @@ <h1 class="govuk-heading-l">What happens next<h1>
 
         <h2 class="heading-xmedium">Shortlist</h2>
 
-        <p>When the opportunity closes, the buyer will score your evidence. If you’re one of the top {{ brief.get('numberOfSuppliers') }} suppliers, you'll go through to the evaluation stage.</p>
-        <p>The buyer will tell you if you're not successful.</p>
+        <p class="govuk-body">When the opportunity closes, the buyer will score your evidence. If you’re one of the top {{ brief.get('numberOfSuppliers') }} suppliers, you'll go through to the evaluation stage.</p>
+        <p class="govuk-body">The buyer will tell you if you're not successful.</p>
 
         <h2 class="heading-xmedium">Evaluation</h2>
 
         <div class="explanation-list">
-          <p class="lead">At the evaluation stage, the buyer will ask you to provide:</p>
-          <ul class="list-bullet">
+          <p class="lead govuk-body">At the evaluation stage, the buyer will ask you to provide:</p>
```

Comment:
Instances of class `lead` should be replaced with `govuk-body-l`

```suggestion
          <p class="lead govuk-body">At the evaluation stage, the buyer will ask you to provide:</p>
```

Reply:
thanks, I made the changes in the last commit

Reply:
If you could squash that last commit into the previous one changing paragraphs that would be great